profile/README.md

```diff
@@ -0,0 +1,138 @@
+        </a>
+        <a href="https://www.w3.org/Style/CSS/Overview.en.html">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/css3/css3-original.svg" alt="CSS" height="50">
+        </a>
+        <a href="https://www.mysql.com/fr/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/mysql/mysql-original.svg" alt="MySQL" height="50">
+        </a>
+        <a href="https://mui.com/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/materialui/materialui-plain.svg" alt="Mui" height="50">
+        </a>
+        <a href="https://azure.microsoft.com/fr-fr">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/azure/azure-original.svg" alt="Azure" height="50">
+        </a>
+        <a href="https://www.terraform.io/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/terraform/terraform-original.svg" alt="Terraform" height="50">
+        </a>
+        <a href="https://eslint.org/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/eslint/eslint-original.svg" alt="ESLint" height="50">
+        </a>
+    </details>
+</div>
+
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Puff.svg" alt="Correlatio logo" height="50"/>
+    <details>
+        <summary>
+            <h3>
+                Correlatio
+            </h3>
+        </summary>
+        <div align="left">
+            <p>
+                Correlatio offer large amount of statistical computation using AI based on open data of French government and other open statistics foundation.
+            </p>
+        </div>
+        <a href="">
+            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Crimen.svg" alt="Crimen logo" height="50">
+        </a>
+    </details>
+</div>
+
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Puff.svg" alt="Libra logo" height="50"/>
+    <details>
+        <summary>
+            <h3>
+                Libra
+            </h3>
+        </summary>
+        <div align="left">
+            <p>
+                Libra offers abundant free and open digital solutions like softwares, websites, servers, templates etc. Open products :
+            </p>
+        </div>
+        <a href="https://github.com/Libra-foundation/Collaboratio">
+            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Collaboratio.svg" alt="Collaboratio logo" height="50">
+        </a>
+        <a href="https://github.com/Libra-foundation/react-template">
+            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/ReactTemplate.svg" alt="React template logo" height="50">
+        </a>
+        <a href="https://github.com/Libra-foundation/python-template">
```

Comment:
The UCS is missing from the list.

Reply:
Yes, that's normal because the project is currently empty. I will add it during a refactor.